AmplifyTools/AmplifyCLIiOS/Sources/AmplifyCLIiOS/Core/Commands/AmplifyCommandTask.swift

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+typealias AmplifyCommandTaskResult = Result<String, AmplifyCommandError>
```

Comment:
Just noticing this: is `String` the output of the task? Would it combine both `stderr` and `stdout` (if run in a shell), or is it up to the task to determine what to put in there? Basically I'm wondering if there's an opportunity to be more explicit about what we're expecting a task result to look like, but only if it's immediately obvious: we can complicate it later if we find a need.

Reply:
It's completely up to the task what to put in there and commands executors (CLI, UI etc) can decide how to use it. 
I thought about using a more explicit type but I couldn't find an obvious replacement.
A possibility would be to also provide more context about the success (i.e. successful but with warnings) but I can't think of an actual usage at this point, so it's probably better to keep it simple for now and complicate it later if needed

Reply:
Yep, that's the right decision for now, thanks.